cookbooks/bcpc-hadoop/libraries/utils.rb

```diff
@@ -481,3 +490,20 @@ def update_oozie_sharelib(host)
     Chef::Log.info("Sharelibupdate: Oozie server not running on #{host}")
   end
 end
+
+def cluster_nodes
```

Comment:
Isn't this duplicated by existing helpers from Lev's PR?

Reply:
This must have been a merge turd as nothing calls it; removed